.travis.yml

```diff
@@ -17,8 +17,8 @@ python:
 before_install:
     - git config --global user.email "OpenStack_TravisCI@f5.com"
     - git config --global user.name "Travis F5 Openstack"
-    - echo "$DOCKERHUB_TOKEN" | docker login -u "$DOCKERHUB_ID" --password-stdin
-    - docker pull f5devcentral/containthedocs:latest
+    - git clone https://github.com/zhang-shengping/docker-images.git  
+    - docker load -i docker-images/f5_agent_travis_images.tar.gz  
```

Comment:
I will recommend to use the docker  image name and tag to name the tarballs, so that others can understand which docker images are required easily.